tri.js

```diff
@@ -1,16 +1,56 @@
 /* Triangle Game Javascript */
 
-var circleColor = "green";
+// variables
+var circleColorOne = "green";
+var circleArrayGreen = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+var circleArrayBlack = [];
 var resetButton = document.querySelector("#reset");
 var circleOne = document.querySelector("#one");
+var allCircles = document.querySelectorAll(".circle");
 
+//start - reset toggle button
+resetButton.addEventListener("click", function () {
+   if (circleColorOne === "green") {
```

Comment:
Probably better to have this as a state variable than to use the color of a page element to determine if the game has started e.g. `let gameStarted = false`

Reply:
Added state variable;  removed color to determine if game started.